app.go

```diff
@@ -60,3 +113,22 @@ func (a *Application) Logger() log.Logger {
 func (a *Application) Statter() stats.Statter {
 	return a.statter
 }
+
+type ApplicationResponse struct {
+	Title  string
+	Text   string
+	Speech string
+	Image  string
+	End    bool
+}
+type Config struct {
+	user string
+}
+type ResponseFunc func(cfg *Config)
```

Comment:
Keep in mind that someone looking at the in a function signature give them zero info as to what is needed by the function. It is not obvious that user is a possibility here.